src/server/api/routes/router.js

```diff
@@ -8,13 +8,49 @@ const userRouter = require("./user.router");
 const authRouter = require("./auth.router");
 const refreshTokenRouter = require("./refresh.token.router");
 const mailRouter = require("./mail.router");
+
+const vesselReportRouter = require("./vessel-reports.router");
+
 const portRouter = require("./port.router");
 const voyagesRouter = require("./voyages.router");
 const suggestedRoutesRouter = require("./suggested-routes.router");
 const organizationRouter = require("./organizations.router");
 const vesselRouter = require("./vessel.router");
+
 const favoriteVesselsRouter = require("./favorite-vessels.router");
 
+// /api/users
+router.use("/users", userRouter);
+
+// /api/auth
+router.use("/auth", authRouter);
+
+// /api/modules
+router.use("/modules", modulesRouter);
+
+router.use("/mail", mailRouter);
+
+router.use("/token", refreshTokenRouter);
+
+// /api/vessel-reports
+router.use("/vessel-reports", vesselReportRouter);
+
+router.use("/ports", portRouter);
+
+// /api/suggested-routes
+router.use("/suggested-routes", suggestedRoutesRouter);
+
+// /api/organizations
+router.use("/organizations", organizationRouter);
+router.use("/vessels", vesselRouter);
+// /api/organizations/organization_id/vessels
+
+// /api/voyages
+router.use("/voyages", voyagesRouter);
+
+// /api/favorite_vessels/
+router.use("/favorite-vessels", favoriteVesselsRouter);
+
```

Comment:
You still did not remove all of the duplicate routes. 

Everything on lines :22 to :53 is the same as :66 to :77.

Only difference being that I add a variable `process.env.API_PATH` before the path. Reason is so we easily can configure the API to run on a different path. F.eks. we may want the first version of the api to run at /api/v1 and later we want to add another api running at /api/v2.

Reply:
removed